sphinx/ext/napoleon/__init__.py

```diff
@@ -256,7 +256,9 @@ def __unicode__(self):
 
         If an entry is just a string, it is interpreted as a header for a generic
         section. If the entry is a tuple/list/indexed container, the first entry
-        is the name of the section, the second is the section key to emulate.
+        is the name of the section, the second is the section key to emulate. If the
+        second entry value is "display_like_parameters" or "display_like_returns", the
+        custom section will be displayed like the parameters section or returns section.
```

Comment:
What about `params_style` and `returns_style`? They could also be all caps like module level constants, or actually be module level constants. Or they could have a `napoleon_`-prefix, which seems the idiomatic way of namespacing special names here.

At any rate, thanks for your interest in this feature, @tk0miya!

Reply:
LGTM for `params_style` and `returns_style`. I think no capitals are needed because of configurations, not code. `napoleon_` prefix is also not needed.